syncopy/datatype/continuous_data.py

```diff
@@ -183,38 +183,25 @@ def time(self):
 
     # # Helper function that reads a single trial into memory
     # @staticmethod
-    # def _copy_trial(trialno, filename, dimord, sampleinfo, hdr):
+    # def _copy_trial(trialno, filename, dimord, sampleinfo):
```

Comment:
Looks all nice and clean, just wondering if we need that commented-in function at all..

Reply:
most likely not. we can remove it in the outer branch, I guess.